group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupMember.java

```diff
@@ -545,6 +547,26 @@ public String currentAssignmentSummary() {
             ')';
     }
 
+    public ConsumerGroupDescribeResponseData.Member asConsumerGroupDescribeMember() {
+        return new ConsumerGroupDescribeResponseData.Member()
+            .setMemberEpoch(memberEpoch)
+            .setMemberId(Uuid.fromString(memberId))
+            .setAssignment(new ConsumerGroupDescribeResponseData.Assignment()
+                .setTopicPartitions(
+                    assignedPartitions.entrySet().stream().map(
+                        item -> new ConsumerGroupDescribeResponseData.TopicPartitions()
+                            .setTopicId(item.getKey())
+                            .setPartitions(new ArrayList<>(item.getValue()))
```

Comment:
Do we also need to set the topicName here? If so, how can I get it from the topicId?

Reply:
Alright, then I let this as is.